vcx/libvcx/src/utils/logger.rs

```diff
@@ -140,7 +140,7 @@ impl LibvcxDefaultLogger {
 
         // ensures that the test that is calling this wont fail simply because
         // the user did not set the RUST_LOG env var.
-        let pattern = Some(env::var("RUST_LOG").unwrap_or("debug".to_string()));
+        let pattern = Some(env::var("RUST_LOG").unwrap_or("trace".to_string()));
```

Comment:
Trace default?

Reply:
Yes, most of the vcx statements are trace so that should be the default.